src/coreclr/tools/superpmi/superpmi-shared/methodcontext.h

```diff
@@ -1218,6 +1218,7 @@ enum mcPackets
     Packet_NotifyInstructionSetUsage = 229,
     Packet_GetAsyncInfo = 230,
     Packet_GetAsyncResumptionStub = 231,
+    Packet_GetHelperFtn = 233,
```

Comment:
I don't think this is needed?

Reply:
My understanding is that if you change the size of contents of the data packet in SPMI you need a new magic number. See MethodContext::MethodInitHelper. Since this PR changes the definition of what the LWM map for GetHelperFtn is, I updated the magic value.

Reply:
It's not needed -- the JIT-EE GUID update is sufficient.